go.mod

```diff
@@ -106,276 +106,16 @@ require (
 )
 
 require (
-	cloud.google.com/go/bigtable v1.5.0 // indirect
```

Comment:
These lines should not be removed. You may not be running `go mod tidy` properly.